samples/apps/bookinfo/rules/mixer-rule-standard-attributes.yaml

```diff
@@ -47,8 +47,30 @@ spec:
         valueType: TIMESTAMP
       source.uid:
         valueType: STRING
+      source.user:
+        valueType: STRING
       target.uid:
         valueType: STRING
+      destination.uid:
+        valueType: STRING
+      connection.id:
+        valueType: STRING
+      connection.received.bytes:
+        valueType: INT64
+      connection.received.bytes_total:
+        valueType: INT64
+      connection.sent.bytes:
+        valueType: INT64
+      connection.sent.bytes_total:
+        valueType: INT64
+      connection.duration:
+        valueType: DURATION
+      context.protocol:
+        valueType: STRING
+      context.timestamp:
```

Comment:
the casing is all over place `serviceAccount`, `user-agent` and `bytes_total`?

Reply:
Those I have just copied over from the API repo, let's see if we can fix it there.

Reply:
Can you point me to the source of truth? I'll file an issue.

Reply:
we need a pass at this anyway in istio/api.